projects/igniteui-angular/src/lib/grids/columns/column.component.ts

```diff
@@ -405,23 +405,40 @@ export class IgxColumnComponent implements AfterContentInit, OnDestroy, ColumnTy
     }
 
     /**
-     * @hidden
+     * Emitted when the column is hidden or shown.
+     *
+     * ```html
+     * <igx-column (hiddenChange)="hiddenChange($event)">
+     * </igx-column>
+     * ```
+     *
+     * @memberof IgxColumnComponent
```

Comment:
Also really no point to this tag, but that's a global issue to clear out so can be ignored.